dynamic.tex

```diff
@@ -382,7 +398,7 @@ \section{Multiple Files}\label{s:dynamic-multiple}
   </head>
   <body>
     <h1>Hello Separate</h1>
-    <div id="app"></div>
+    <div id="app" type="text/babel"></div>
```

Comment:
Why does this `div` need to have `type="text/babel"`? There's no JavaScript inside this `div` that needs to be translated.